src/drop-down-icon.jsx

```diff
@@ -112,7 +112,7 @@ const DropDownIcon = React.createClass({
           <div onTouchTap={this._onControlClick}>
               <FontIcon
                 className={iconClassName}
-                style={this.prepareStyles(iconStyle)}>{this.props.iconLigature}</FontIcon>
+                style={iconStyle}>{this.props.iconLigature}</FontIcon>
```

Comment:
In other words, all flipping and prefixing should only be done before passing the style object to a dom element.